### DISCUSSION TITLE
Please add OPENVPN_PORT variable for custom port

### DISCUSSION THREAD

**Discussion by cacti-user** [qdm12/gluetun#956] [Ideas]
Hi there,

Is it possible to select a custom port rather than standard 1194 UDP? Can we add an OPENVPN_PORT variable? For example, Privado VPN allows 443, 8443 and 8080 UDP and TCP in addition to 1194.

In the past, I have had issues where 1194 UDP stopped working with Privadovpn so had to use another port but can not do this via standard env. variables. So I had to use a custom OpenVPN profile.

(Would also be great if you can add the PrivadoVPN TCP option)
![image](https://user-images.githubusercontent.com/68411536/164132786-4804936b-3364-4879-bcc4-a584ed244528.png)

**Comment by qdm12** [comment]
Thanks for pointing this out. This was a bug really :+1: 

It's done in the latest image in 0c0f1663b1bc27bb4eb32073f3f762c9d2b84239 (through another refactor as a side effect luckily!).
I'll be part of the next release v3.29.0.

**Reply by cacti-user** [reply]
That's brilliant thanks mate 🙏
